ee/api/test/test_feature_flag.py

```diff
@@ -0,0 +1,25 @@
+from ee.api.test.base import APILicensedTest
+from ee.models.organization_resource_access import OrganizationResourceAccess
+from ee.models.role import Role, RoleMembership
+from posthog.models.feature_flag import FeatureFlag
+from posthog.models.organization import OrganizationMembership
+
+
+class TestFeatureFlagEnterpriseAPI(APILicensedTest):
+    def test_adding_role_edit_access_is_not_restrictive(self):
```

Comment:
wait, where did all the other tests for permissions go? 😅

Reply:
It's still where it's at originally, this is a new `ee feature flags` test file since it has to do with feature flags on EE 😃 Also I needed it to be a licensed API test